joint_trajectory_controller/doc/userdoc.rst

```diff
@@ -224,18 +335,6 @@ gains.<joint_name>.normalize_error (bool)
 
   Default: false
 
-ROS2 interface of the controller
```

Comment:
I would keep this here

Reply:
I would actually extend this to be more like other new controllers, describing all interfaces in detail.

Reply:
I applied the structure of the PID controller, what do you think?